### PR TITLE
Milestone 1: React-calculator

### PR DESCRIPTION
**Pull request by Mesi21**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,6 @@
-import React from 'react';
```
Comment (azdrenymeri) [inline src/index.js:1]: You need  to import `React` otherwise the `jsx` written inside this file will not be recognized

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,6 @@
...
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import './index.css';
+import ReactDOM from 'react-dom';
+import App from './components/App';
```
Comment (azdrenymeri) [inline src/index.js]: The file `./components/App` doesn't exist

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @Mesi21  :wave: 
The project is set up correctly except there are some errors on your code.
#### Kindly fix the issued mentioned on the comments and add it for a re-review

File: src/index.js (hunk 1 above)
Comment (Mesi21) [reply src/index.js:1]: Thank you very much. I fixed it.

File: .stickler.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+linters:
+  flake8:
+  eslint:
+    config: './eslint.config'
+  stylelint:
+files:
+  ignore:
+    - '.eslintrc.json'
```
Comment (bolah2009) [inline .stickler.yml]: Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/react-redux).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	  ],
+	  "space-before-function-paren": [
+		"error",
+		{ "anonymous": "always", "named": "never", "asyncArrow": "always" }
+	  ],
+	  "indent": ["error", 2],
+	  "no-trailing-spaces": ["error", { "skipBlankLines": true }],
+	  "no-console": ["error", { "allow": ["warn"] }]
+	}
+  }
```
Comment (bolah2009) [inline .eslintrc.json:22]: Please let's use the correct config. See the previous comment

File: eslint.config [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+		"import/extensions": "off",
+		"arrow-parens": [2, "as-needed"],
+		"import/no-named-as-default": 0,
+		"no-named-as-default-member": 0,
+		"no-nested-ternary": 0,
+		"no-unused-expressions": 0,
+		"no-restricted-globals": 0,
+		"import/no-unresolved": "off"
+    }
+}
```
Comment (bolah2009) [inline eslint.config:74]: Please, let delete this

**Review by bolah2009 — APPROVED** [review]
Hi @Mesi21 

@azdrenymeri seems to miss this but you need a descriptive PR description and use the corrct config from [the linter repo](https://github.com/microverseinc/linters-config/tree/master/react-redux). Please see [this pathwright lesson for detailed instructions](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

> Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next milestones. :star2: 

Please see the comment below and update PR description before merging.

Cheers
[Bola](https://bolabuari.com/)

**Comment by Mesi21** [comment]
Solved all stickler errors and changes requested by TSE

File: .stickler.yml (hunk 3 above)
Comment (Mesi21) [reply .stickler.yml]: Thank you for your observations. I've made the necessary changes.

**Pull request merged**